app/src/app/[lng]/settings/page.tsx

```diff
@@ -51,6 +48,26 @@ export default function Settings({
 
   const { t } = useTranslation(lng, "settings");
 
+  const { data: userInfo, isLoading: isUserInfoLoading } =
+    api.useGetUserInfoQuery();
+
+  const { data: currentUserData, isLoading: isUserDataLoading } =
+    api.useGetUserQuery(
+      {
+        locode: userInfo?.defaultCityLocode!,
+        userId: userInfo?.userId!,
+      },
+      { skip: !userInfo },
+    );
+
+  const { data: cityUsers } = api.useGetCityUsersQuery(
+    { locode: userInfo?.defaultCityLocode! },
+    {
+      skip: !userInfo,
+      pollingInterval: 3000,
```

Comment:
Should we really poll this API route, or is it sufficient to use `invalidateTags` in the mutations to force a reload of the data?